src/lib/INPUT/input.cpp

```diff
@@ -0,0 +1,81 @@
+#ifdef TARGET_AXIS_THOR_2400_TX
+#include "input.h"
```

Comment:
I feel like `input.h .cpp` is way to generic.  It should be something more descriptive of the hardware.

Reply:
The input is for 5-way buttons, In fact the input function's API is generic and only GPIO pin define maybe target specific.

Reply:
5waybutton is a better name and more descriptive of whats inside the file.

Reply:
update it at d617ecf